Iceberg-Plugin-Pharo/IcePharoNewBranchFromIssuePanel.class.st

```diff
@@ -84,19 +84,19 @@ IcePharoNewBranchFromIssuePanel >> initializeWidgetsContents [
 	self initializeCurrentBranchLabel.
 	branchLabel := self newLabel label: 'Pharo issue Number'.
 	issueNumberText := self newTextInput
-		ghostText: 'e.g., 12345';
+		placeholder: 'e.g., 12345';
 		autoAccept: true.
 	issueLabel := self newLabel label: 'Title'.
 	issueText := self newTextInput
-		ghostText: 'e.g., a-fogbugz-issue';
+		placeholder: 'e.g., a-fogbugz-issue';
 		autoAccept: true.
-		
 	issueNumberText whenBuiltDo: [ :w | w widget wrapFlag: false ].
 	issueText
 		whenBuiltDo: [ :w | 
 			w widget wrapFlag: false.
 			w widget enabled: false ].
-	issueNumberText textHolder whenChangedDo: [ :text | self validateIssue: text ]
+	issueNumberText textHolder
```

Comment:
You should not use textHolder but whenTextChangedDo:

Reply:
Shouldn't these holders be deprecated and rewritten in spec with spec observable slots?

Reply:
Those accessors to holders are already deprecated in Spec2 since classes using presenters should not use the internals of the presenters when the API already provides a way to do what the users want. 

There is not an automatic rewrite because what should be replaced will depend on what the user did with the holder.

Reply:
ok! I wonder why the did not catch the deprecation then...